### PR TITLE
Update build instructions for gcc 11, and 10.3 for Aarch64 Linux

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

File: doc/build-instructions/Build_Instructions_V11.md [hunk 1]
```diff
@@ -101,11 +101,11 @@ apt-get update \
     ...
 ```
 
-2. The previous step installed g++-7 and gcc-7 packages, which might be different
+2. The previous step installed g++-11.2 and gcc-11.2 packages, which might be different
```
Comment (keithc-ca) [inline doc/build-instructions/Build_Instructions_V11.md:104]: This is not true. ubuntu:16.04 is no longer supported by `mkdocker.sh`, ubuntu:18.04 would yield gcc-8 and ubuntu:20.04 would yield gcc-10. gcc-11 is only included for centos.

I have changes to `mkdocker.sh` that would support ubuntu:22.04 with gcc-11, but I don't have access to POWER 9 hardware to test it fully (POWER 8 hardware is not supported by 22.04).

File: doc/build-instructions/Build_Instructions_V11.md [hunk 2]
```diff
@@ -727,7 +727,7 @@ bash get_source.sh
 
 You must install a number of software dependencies to create a suitable build environment on your AArch64 Linux system:
 
-- GNU C/C++ compiler (The Docker image uses GCC 7.5)
+- GNU C/C++ compiler 10.3 (The Docker image uses GCC 7.5)
```
Comment (keithc-ca) [inline doc/build-instructions/Build_Instructions_V11.md:730]: I'm not sure where the reference to 7.5 came from; I don't think it's currently accurate.

**Review by keithc-ca — CHANGES_REQUESTED** [review]
Comments apply to all versions.

File: doc/build-instructions/Build_Instructions_V11.md (hunk 2 above)
Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V11.md:730]: I assume it's referring to the alinux docker file
https://github.com/eclipse-openj9/openj9/blob/master/buildenv/docker/aarch64-linux_CC/Dockerfile#L44

File: doc/build-instructions/Build_Instructions_V11.md (hunk 1 above)
Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V11.md:104]: Does this help?
https://hyc-runtimes-jenkins.swg-devops.com/label/hw.arch.ppc64le.p9/

File: doc/build-instructions/Build_Instructions_V11.md (hunk 2 above)
Comment (keithc-ca) [reply doc/build-instructions/Build_Instructions_V11.md:730]: That makes sense. Thanks.

File: doc/build-instructions/Build_Instructions_V11.md (hunk 1 above)
Comment (keithc-ca) [reply doc/build-instructions/Build_Instructions_V11.md:104]: I'll give that a try.

Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V11.md:104]: I expect `/usr/local/gcc11/bin/gcc-11.2` only works on centos, what is the value to use for Ubuntu?

Comment (keithc-ca) [reply doc/build-instructions/Build_Instructions_V11.md]: On Ubuntu:22.04 it's 11.4 (currently).

Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V11.md]: Hopefully the update is good now.

File: doc/build-instructions/Build_Instructions_V11.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ Now that you have the Docker image running, you are ready to move to the next st
 If you don't want to user Docker, you can still build directly on your Ubuntu system or in a Ubuntu virtual machine. Use the output of the following command like a recipe card to determine the software dependencies that must be installed on the system, plus a few configuration steps.
 
 ```
-bash mkdocker.sh --tag=openj9 --dist=ubuntu --version=16.04 --gitcache=no --jdk=11 --print
+bash mkdocker.sh --tag=openj9 --dist=ubuntu --version=22 --gitcache=no --jdk=11 --print
```
Comment (keithc-ca) [inline doc/build-instructions/Build_Instructions_V11.md:94]: All sample uses of `mkdocker.sh` should be updated to say `--version=22`, e.g. line 76 of this file.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: doc/build-instructions/Build_Instructions_V11.md (hunk 3 above)
Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V11.md:94]: Thanks, updated it.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**